app/src/main/res/values/strings.xml

```diff
@@ -300,8 +300,9 @@
     <!-- Preference Summary: Biometrics (iris scan, fingerprint, or facial scan) can be used to unlock Focus if the app had a URL
         open when the app was last run to ensure that only the phone's owner can view a previous session.
         %1$s will be replaced by the app name (e.g. Firefox Focus) -->
-    <string name="preference_security_biometric_summary">Your fingerprint can unlock %1$s if a URL is already opened in the app.
+    <string name="preference_security_biometric_summary" moz:removedIn="102" tools:ignore="UnusedResources">Your fingerprint can unlock %1$s if a URL is already opened in the app.
     Stealth will be turned on.</string>
+    <string name="preference_security_biometric_summary2">Unlock using fingerprint if you’ve added Shortcuts or when a website is already open in %1$s.</string>
```

Comment:
Looking at the screenshot posted by Jeff there is no punctuation mark at the end of these strings.
Maybe we should check with Jeff on the general approach to be used but in the meantime since all strings have a dot a the end I think it's okay to use one here also.

Looking at just the string it may not be clear what "Shortcuts" refer to. Can you update the string comment to also offer more details about the "Shortcuts" feature?

nit: There is no need for the `%1` positional argument if there is only one argument this string takes. `%s` should be enough.